### DISCUSSION TITLE
Vid "PyTorch Fundamentals" - What is "ndim" for tensor?

### DISCUSSION THREAD

**Discussion by sunnysaw07** [mrdbourke/pytorch-deep-learning#1164] [General]
Hello Everyone,
I'm just checking

Code: 

```python
import torch
torch.randn(3,1)
```

**Comment by pem131** [comment]
Hello,

`tensor.ndim` returns the number of dimensions of the tensor (you can also use `tensor.dim()`)

in your case:

`torch.randn(3,1).ndim`

would give you 2, the number of dimensions.

Best wishes.
